installer_linux/make_deb.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# preflight check
+
+if [[ ! -f ./make_deb.sh ]]; then
+	echo "You must run this script from within the installer_osx directory!"
+	exit 1
+fi
+
+# version
+
+if [ "$SURGE_VERSION" != "" ]; then
+    VERSION="$SURGE_VERSION"
+elif [ "$1" != "" ]; then
+    VERSION="$1"
+fi
+
+if [ "$VERSION" == "" ]; then
+    echo "You must specify the version you are packaging!"
+    echo "eg: ./make_deb.sh 1.0.6b4"
+    exit 1
+fi
+
+# Valid version for DEB start with a digit.
+DEB_VERSION=$VERSION
+if [[ ${DEB_VERSION:0:1} =~ [0-9] ]]; then
+    echo "DEB VERSION same as Version"
+else
+    DEB_VERSION="0.${VERSION}"
+    echo "DEB VERSION is ${DEB_VERSION}"
+fi
+
+
+# Names
+SURGE_NAME=Surge
+# SURGE_NAME=surge-synthesizer
+
+# locations
+
+mkdir -p surge-synthesizer/usr/lib/vst
+mkdir -p surge-synthesizer/usr/lib/vst3
+mkdir -p surge-synthesizer/usr/share/${SURGE_NAME}/doc
+mkdir -p surge-synthesizer/DEBIAN
+
+# build control file
+
+if [[ -f surge-synthesizer/DEBIAN/control ]]; then
+	rm surge-synthesizer/DEBIAN/control
+fi
+touch surge-synthesizer/DEBIAN/control
+cat <<EOT >> surge-synthesizer/DEBIAN/control
+Package: surge-synthesizer
```

Comment:
package name should be just "surge", no?

Reply:
Yup changed this in my force push now.